public/vocables/js/search/elasticSearchProxy.js

```diff
@@ -4,6 +4,9 @@ var ElasticSearchProxy = (function () {
 
     self.queryElastic = function (query, indexes, callback) {
         if (!indexes) indexes = context.curentSearchIndexes;
+        if (!Array.isArray(indexes)) {
+            indexes = [indexes];
+        }
```

Comment:
if the idea is to make sure indexes is an Array, let's instead throw an exception if that is not the case.